packages/cxl-lumo-styles/scss/themes/vaadin-button.scss

```diff
@@ -21,3 +22,31 @@
 :host([theme="linkedin"]:hover)::before {
   opacity: 0.05;
 }
+
+:host([theme~="primary"]),
+:host([theme~="secondary"]) {
+  border: 2px solid var(--lumo-contrast);
+  border-radius: 10px;
+  box-shadow: 3px 4px 0 var(--lumo-contrast);
+  font-size: var(--lumo-font-size-l);
+  font-weight: 600;
+  padding: var(--lumo-space-m) var(--lumo-space-xl) !important;
+}
+
+:host([theme~="contrast"][theme~="primary"]) {
+  background-color: var(--cxl-b2b-redesign-blue) !important;
```

Comment:
Why this needs to be `!important`? Have you tried approach used for Linkedin button?

```css
:host([theme="linkedin"]) {
  --_lumo-button-color: #fff;
  --_lumo-button-background-color: #0073b1;
}
```

Reply:
I remember trying that, and it got overridden somewhere. But I'll double-check at one point, I'll try to keep in mind.